.github/workflows/kind-integration.yml

```diff
@@ -0,0 +1,90 @@
+name: KinD DSPO/DSP Integration Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - controllers/**
+      - api/**
+      - config/**
+      - tests/**
+    types:
+      - opened
+      - reopened
+      - closed
+      - synchronize
+
+concurrency:
+  group: ${{ github.head_ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+env:
+  IMAGE_REPO_DSPO: data-science-pipelines-operator
+  QUAY_ORG: hukhan
```

Comment:
I'm assuming this is a workaround for executor permissions or what not.  Do we have a ticket to migrate this to the opendatahub org?

Reply:
those fields should not have been there, removed